docs/fundamentals/diagnostics.rst

```diff
@@ -1,6 +1,146 @@
+	:emphasize-lines: 2,21
+
+The above code, which is built from the ASP.NET 5 Empty template, includes a simple mechanism for creating an exception on line 36. If a request includes a non-empty querystring parameter for the variable ``throw`` (e.g. a path of ``/?throw=true``), an exception will be thrown. Line 21 makes the call to ``UseErrorPage()`` with `ErrorPageOptions <https://github.com/aspnet/Diagnostics/blob/dev/src/Microsoft.AspNet.Diagnostics/ErrorPageOptions.cs>`_ set to ``ShowAll``. Using ``ErrorPageOptions`` you can toggle the visibility of the following features of the error page:
+
+- Exception details
+- Source code
+- Query
+- Cookies
+- Headers
+- Environment
+
+To disable the display of cookies, for instance, you would pass in an options instance configured as follows:
+
+.. code-block:: c#
+
+	var errorPageOptions = new ErrorPageOptions();
+	errorPageOptions.ShowCookies = false;
+	app.UseErrorPage(errorPageOptions);
+
+Notice that the call to ``UseErrorPage()`` is wrapped inside an ``if`` condition that checks the current ``EnvironmentName``. This is a good practice, since you typically do not want to share detailed diagnostic information about your application publicly once it is in production. This check uses the ``ASPNET_ENV`` environment variable. If you are using Visual Studio 2015, you can customize the environment variables used when the application runs in the web application project's properties, under the Debug tab, as shown here:
+
+.. image:: diagnostics/_static/project-properties-env-vars.png
+	
+Setting the ``ASPNET_ENV`` variable to anything other than Development (e.g. Production) will cause the application not to call ``UseErrorPage()``, and thus any exceptions will be handled by the underlying web server (in this case, IIS Express) as shown here:
+
+.. image:: diagnostics/_static/oops-500.png
+
+We will cover the features provided by the error page in the next section (ensure ``ASPNET_ENV`` is reset to Development if you are following along).
+
+Using the error page during development
+---------------------------------------
+
+The default error page will display some useful diagnostics information when an unhandled exception occurs within the web processing pipeline. The error page includes several tabs with information about the exception that was triggered and the request that was made. The first tab shows the stack trace:
+
+.. image:: diagnostics/_static/errorpage-stack.png
+
+The next tab shows the contents of the Querystring collection, if any:
+
+.. image:: diagnostics/_static/errorpage-query.png
+
+In this case, you can see the value of the ``throw`` parameter that was passed to this request. This request didn't have any cookies, but if it did, they would appear on the Cookies tab. You can see the headers that were passed, here:
+
+.. image:: diagnostics/_static/errorpage-headers.png
+
+Finally, any environment variables defined for the server environment would be displayed on the Environment tab.
```

Comment:
Remove any reference to the environment tab, it's a leftover from Katana referring to the OWIN environment, not environment variables. https://github.com/aspnet/Diagnostics/issues/135